templates/huawei_vrp_display_version.textfsm

```diff
@@ -0,0 +1,9 @@
+Value VRP_VERSION (\S+)
+Value PRODUCT_VERSION (.+)
+Value MODEL (.+)
+Value UPTIME (.+)
+
+
+Start
+  ^.*software,\sVersion\s${VRP_VERSION}\s\(${PRODUCT_VERSION}\)

```

Comment:
Do you mind changing the each `\s` to `\s+` just in case they change any whitespace between versions and it won't break the template.

Reply:
done